OpenSim/Simulation/TableProcessor.h

```diff
@@ -183,19 +183,36 @@ class OSIMSIMULATION_API TabOpLowPassFilter : public TableOperator {
     OpenSim_DECLARE_PROPERTY(cutoff_frequency, double,
             "Low-pass cutoff frequency (Hz) (default is -1, which means no "
             "filtering).");
-    TabOpLowPassFilter() { constructProperty_cutoff_frequency(-1); }
+    OpenSim_DECLARE_PROPERTY(trim_to_original_time_range, bool,
+            "Trim the rows of the output table to match the original table's "
+            "time range after padding and filtering (default: true).");
```

Comment:
Perhaps delete "after padding" as I don't think the user-facing documentation explains padding (what/why).